wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java

```diff
@@ -66,6 +66,11 @@ public ProtoAdapter(FieldEncoding fieldEncoding, Class<?> javaType) {
     return new RuntimeEnumAdapter<>(type);
   }
 
+  /** Returns the default adapter for the type of {@code Message}. */
```

Comment:
Default? I don't think we use that term anywhere. I think just "the adapter" works.